sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_service_client_helper.py

```diff
@@ -0,0 +1,28 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=docstring-keyword-should-match-keyword-only
+
+from typing import TYPE_CHECKING
+from urllib.parse import urlparse
+
+if TYPE_CHECKING:
+    from urllib.parse import ParseResult
+
+
+def _parse_url(account_url: str) -> "ParseResult":
```

Comment:
Move this to `_parser` or merge with the next PR

Reply:
What does this comment mean?

Reply:
Is the exact same function going to be used by multiple clients or is it different per client? If its the same for multiple clients, moving it to parsers makes sense. As for which PR, whatever works for you.